vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/JavascriptActions.java

```diff
@@ -37,12 +41,10 @@
     private final IWebDriverProvider webDriverProvider;
     private final IWebDriverManager webDriverManager;
 
-    private final ThreadLocal<String> userAgent = ThreadLocal.withInitial(
-        () -> executeScript("return navigator.userAgent"));
-
-    private final ThreadLocal<Double> devicePixelRatio = ThreadLocal.withInitial(() -> {
-        Number devicePixelRatio = executeScript("return window.devicePixelRatio");
-        return devicePixelRatio.doubleValue();
+    private final ThreadLocal<BrowserConfig> config = ThreadLocal.withInitial(() -> {
+        String info = executeScript("return JSON.stringify({userAgent: navigator.userAgent,"
+                + "devicePixelRatio: window.devicePixelRatio})");
+        return new Gson().fromJson(info, BrowserConfig.class);
```

Comment:
why not to use JsonUtils? or even having map you can just get the values from map by keys